packages/python-bindep/python-bindep.spec

```diff
@@ -74,6 +70,9 @@ set -ex
 
 
 %changelog
+* Tue Dec 12 2023 Patrick Creech <pcreech@redhat.com> - 2.11.0-5
+- Rollback overzealous obsoletes
```

Comment:
bindir!

Reply:
See previous comment

Reply:
per irc convo, we're gonna keep the Obsoletes on anything with bindir for now, and can remove as we find need to.

Reply:
removed